PokeDemo/Controllers/HomeViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class HomeViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var pokemonTableView: UITableView!
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+
+    private let presenter = HomeViewPresenter(service: Service.shared)
+    private var pokemonList: [Pokemon]? = [Pokemon]()
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        pokemonTableView.register(UINib(nibName: PokemonTableViewCell.cellIdentifier(), bundle: nil), forCellReuseIdentifier: PokemonTableViewCell.cellIdentifier())
```

Comment:
Can be renamed as `PokemonListCell`

Reply:
Done in ec81c95097ca23d12d71ba906f27718255168054